src/components/interface/SwapForm/components/Leverage/Leverage.tsx

```diff
@@ -44,49 +46,14 @@ const Leverage = ({
     isUndefined(internalValue);
   const timer = useRef<number>();
 
-  const maxNotional = !isDisabled ? Math.min(notional, availableNotional) : 10;
```

Comment:
seems like we removed the usages of the variable isDisabled. IMO it is now unused variable. Can you delete it?